### DISCUSSION TITLE
Error: Pinecone doesn't work Flowise

### DISCUSSION THREAD

**Discussion by Mariahwo** [FlowiseAI/Flowise#1473] [Q&A]
When I paste my API conclusions from Pinecone into Flowise, I get the following error:

A call to
https://gcp-starter-gnqojri.svc.gcp-starter.pinecone.io/query
returned HTTP status 404.

Missing request header "Api-Key"

I have added the environment and the API key correctly and in previous projects it has also worked with Flowise

I have now contacted Pinecone support as well as you because I don't know where the error is coming from. 

Can you explain to me why this could be, what possibilities there are or whether this is just a bug?

**Comment by HenryHengZJ** [comment]
can you provide a screenshot or export of your flow? have you created Pinecone credential on Flowise?

**Comment by PingUmich** [comment]
> can you provide a screenshot or export of your flow? have you created Pinecone credential on Flowise?

Hi Henry, Could you take a look my case, #1595 ? I had the same problem, and I attached a screenshot. Thank you so much!
